### DISCUSSION TITLE
cn.hutool.core.lang.Holder在5.8.20中没有了

### DISCUSSION THREAD

**Discussion by championcp** [looly/hutool#3245] [General]
请问在5.4.2中的cn.hutool.core.lang.Holder在5.8.20中用什么替代了呢？

**Comment by looly** [comment]
![image](https://github.com/dromara/hutool/assets/1432613/cfaa27be-827b-41f4-8a6c-6d20f616bf7b)

使用`cn.hutool.core.lang.mutable.MutableObj`

**Comment by championcp** [comment]
感谢您的回复~☺
